pipelines/resources/deploy-ztp-spokes.yaml

```diff
@@ -131,6 +131,66 @@ spec:
       - name: ztp
         workspace: ztp
 
+    # Deploy Disconnected Registry
+  - name: deploy-disconnected-registry-ocp
+    taskRef:
+      name: spoke-deploy-disconnected-registry-spokes-ocp
+    params:
+      - name: spokes-config
+        value: $(params.spokes-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+    runAfter:
+      - deploy-disconnected-registry
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+    # Deploy Disconnected Registry
+  - name: deploy-disconnected-registry-olm
+    taskRef:
+      name: spoke-deploy-disconnected-registry-spokes-olm
+    params:
+      - name: spokes-config
+        value: $(params.spokes-config)
+      - name: kubeconfig
+        value: $(params.kubeconfig)
+      - name: ztp-container-image
+        value: $(params.ztp-container-image)
+      - name: mock
+        value: $(params.mock)
+    runAfter:
+      - deploy-disconnected-registry
+    workspaces:
+      - name: ztp
+        workspace: ztp
+
+    # Deploy Disconnected Registry
```

Comment:
Can you update the comment here?

Reply:
> Since we are parallelizing:
> 
> Is there a reason why the `deploy-ocs` step can't be run in parallel with the `deploy-metallb` one?

@flaper87 the reason is because we don't have API for ocs. Yesterday I tested it just to confirm and the installation was failed